docs/user-guide/downward-api/README.md

```diff
@@ -1,11 +1,11 @@
 Following these examples, you will create a pod with a container that consumes the pod's name,
-namespace, and resource values using the [downward API](http://kubernetes.io/docs/user-guide/downward-api/).
+namespace, and resource values using the [downward API](/docs/user-guide/downward-api/).
```

Comment:
This should be `/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/`